src/en/users/documentation/index.html

```diff
@@ -3,4 +3,31 @@
 order: 2
 ---
 
-<h1 class="h1">{{ title }}</h1>
+<h1 class="h1">{{ title }}</h1><div>
+
+	<section class="section">
+		  <div class="grid lg:grid--cols-2 grid--gap-14 lg:grid--gap-28">
+
+        <p class="standout">
+          The Ceph Foundation believes that all storage problems should be solvable with open-source software.
+        </p>
+	<div class="bg-grey-300 p-5 rounded-2">
+	<p class="p">
+	The upstream Ceph documentation is linked below.
+	</p>
+	<p class="p">
+	The Ceph Install Guide describes how to deploy a Ceph cluster. The
+	cephadm guide describes how to use the cephadm utility to manage your
+	Ceph cluster. If you are consulting the documentation to learn the
+	rules and customs that govern making a pull request against the
+	ceph/ceph Github repository, read the Developer Guide. For more
+	in-depth information about the nature of Ceph, see the Architecture
+	Guide on the page linked below. 
```

Comment:
should `Ceph Install Guide` etc be links?

Reply:
I think if we were on a website in 1998, the answer would be "yes", but in this case we have a button linked to the top-level of the docs and that button appears below this text. I think that adding the links in to this text makes this into a David Foster Wallace novel and takes the whammy out of the button.

If you disagree strongly with this, I'll put the links in. Let me know what you think.